can_tools/scrapers/official/HI/hi_county.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import us
+import requests
+import re
+import json
+from bs4 import BeautifulSoup
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import (
+    TableauDashboard,
+    TableauMapClick,
+    StateDashboard,
+)
+
+from can_tools.scrapers.util import requests_retry_session
+
+
+class HawaiiVaccineCounty(TableauDashboard, StateDashboard):
+    has_location = False
+    source = "https://health.hawaii.gov/coronavirusdisease2019/what-you-should-know/current-situation-in-hawaii/#vaccine"
+    source_name = (
+        "State of Hawai'i - Department of Health"
+        "Disease Outbreak Control Division | COVID-19"
+    )
+    location_type = "county"
+    state_fips = int(us.states.lookup("Hawaii").fips)
+    # https://public.tableau.com/shared/H33ZZ9HCC?:display_count=y&:origin=viz_share_link&:embed=y
+    # Hawai'i https://public.tableau.com/shared/7TCBHC568?:display_count=y&:origin=viz_share_link&:embed=y
+    filterFunctionName = f"county%20select" # this is the name of the user action, not the filter function name. doesn't work
+    baseurl = "https://public.tableau.com"
+    viewPath = "HawaiiCOVID-19-VaccinationDashboard/VACCINESBYCOUNTY"
+    # baseurl = "https://public.tableau.com/shared/"
+    # viewPath = "7TCBHC568"
+
+    
+    def fetch(self):
+        # Get Maui County
+        self.filterFunctionValue = 'MAUI'
```

Comment:
You'll want to use the following values (case-sensitive) for the `self.filterFunctionValue`:

- `Hawaii`
- `Honolulu`
- `Kauai`
- `Maui`

Reply:
Done